### DISCUSSION TITLE
Stuck on "Attendees of this school must verify and use their academic email address." although I verified my university email address

### DISCUSSION THREAD

**Discussion by hello-sabira** [community/community#43568] [GitHub Education]
So I've been trying to Sign Up for Github student developer pack. 
My primary email is my personal one and I verified my university one to get the student developer pack. 
When I went to the Upload proof of your academic status page, I uploaded my Dated student ID and I am receiving this after I clicked Process button. 
"Your application cannot be reviewed until you fix the following:
Attendees of this school must verify and use their academic email address."

How to fix this? From which page?
I'm unsure what to do now. I've reached out to support with proof but I have a strong feeling won't get any reply. 
Anybody else going through same issue?

**Comment by Benhrpr** [comment]
Having the same issue, any luck?
